SocketIOClientSwift/SocketEngine.swift

```diff
@@ -525,8 +527,8 @@ public final class SocketEngine: NSObject, WebSocketDelegate {
 
                 if msg.characters.count != 0 {
                     // Be sure to capture the value of the msg
-                    dispatch_async(handleQueue) {[weak self, msg] in
```

Comment:
Is there a reason you're removing the weak self? Generally when you dispatch async you don't want to create a strong reference